SmartDeviceLink/SDLLockScreenViewController.m

```diff
@@ -70,6 +70,28 @@ - (void)setBackgroundColor:(UIColor *_Nullable)backgroundColor {
     [self sdl_layoutViews];
 }
 
+- (void)setLockedLabelText:(NSString *_Nullable)lockedLabelText {
+    _lockedLabelText = lockedLabelText;
+    
+    [self sdl_layoutViews];
+}
+
+#pragma mark - Swipe Gesture
+
+- (void)addDismissGestureWithCallback:(SwipeGestureCallbackBlock)swipeGestureCallback {
+    self.dismissGestureCallback = swipeGestureCallback;
+    UISwipeGestureRecognizer *swipeGesture = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(sdl_didSwipeToDismiss:)];
+    [swipeGesture setDirection: UISwipeGestureRecognizerDirectionDown];
+    [self.view addGestureRecognizer:swipeGesture];
+}
+
+- (void)removeDismissGesture {
+    self.view.gestureRecognizers = [[NSArray alloc] init];
```

Comment:
In combination with the above change, this should just call `removeGestureRecognizer:` with the stored gesture and then the stored gesture cleared.